test/support/module_with_builtin_type_shadowing.ex

```diff
@@ -1,6 +1,9 @@
-defmodule ElixirSenseExample.ModuleWithBuiltinTypeShadowing do
-  @compile {:no_warn_undefined, {B.Callee, :fun, 0}}
-  def plain_fun do
-    B.Callee.fun()
+# TODO: revert this commit when "warnings-as-errors: true" on CI moves to Elixir 1.11
```

Comment:
This is not really needed. Tests in this project won't be warning free as the code needs to deal with not complete/not correct source

Reply:
Having `MIX_ENV` set to `dev` (or anything but `test`) for `mix compile --warnings-as-errors` indeed ignores this file, but `mix test` will not being able to reuse the compilation. Should I revert this commit and set `MIX_ENV=dev` for this step?

Reply:
No longer necessary

Reply:
Solved in e291b52f498133d558c924a0ceea0abcc3b4818b